Source/Microsoft.Teams.Apps.FAQPlusPlus.Configuration/GlobalSuppressions.cs

```diff
@@ -1,9 +1,6 @@
-﻿
-// This file is used by Code Analysis to maintain SuppressMessage 
-// attributes that are applied to this project.
-// Project-level suppressions either have no target or are given 
-// a specific target and scoped to a namespace, type, member, etc.
+﻿// <copyright file="GlobalSuppressions.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
 
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "Not required to catch specific exception.", Scope = "member", Target = "~M:Microsoft.Teams.Apps.FAQPlusPlus.Configuration.Controllers.HomeController.RefreshQnAMakerEndpointKeyAsync~System.Threading.Tasks.Task{System.Boolean}")]
```

Comment:
@abbodh Please remove this file and move the suppression and justification to the site using #pragma
And please fix the justification, it doesn't quite make sense :(

Reply:
Removed file and addressed the style cop warnings.